config/routes.rb

```diff
@@ -7,7 +7,8 @@
       end
 
       resources :instructors, only: [] do
-        resources :instructor_students, only: :index, as: 'students'
+        resources :instructor_students, only: :index , as: 'students'
+        post '/instructor_students/search', to: 'instructor_students#show'
```

Comment:
Couldn't get this in the resources syntax, any ideas?

Reply:
I believe the POST verb is reserved for the create action when using resources. I think your hand rolled route is correct because it's unrestful. You could try still doing under the Instructor_students resource, but hand rolling POST `/search` to a specific action. In which case, you may consider updating the action on the controller to an unrestful action as well.